src/utils/satelliteUtils.ts

```diff
@@ -19,12 +20,20 @@ export default class SatelliteUtilities {
         return null
     }
 
+    static async getSatellitesByRange(range: Range): Promise<SatelliteDocument[]> {
+        const foundSatellites = await Satellite.find({
+            perigee: { $gte: range.xStart, $lte: range.xEnd },
```

Comment:
This will search for a square area, in the task it's said that the result is supposed to be a satellite(s) 
in a range around a specific position. Which is a circle.